scripts/index.js

```diff
@@ -15,13 +15,21 @@ if (appConfig.features.useTansaProofing) {
 // since tests do not import this file.
 angular.module('superdesk.config').constant('config', appConfig);
 
+// Superdesk modules to be loaded in bootstrap process
```

Comment:
Please undo the changes in this file and remove the `features.planning` item from the configuration. It is no longer needed after https://github.com/superdesk/superdesk-client-core/pull/995.

To enable the `superdesk.planning` module, simply install it with `npm` and add this into `superdesk.config.js`:
```js
apps: ['superdesk-planning']
```